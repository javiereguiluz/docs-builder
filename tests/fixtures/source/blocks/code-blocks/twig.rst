.. code-block:: twig

    {# some code #}
    {%
        set some_var = 'some value' # some inline comment
    %}
    {{
        # another inline comment
        'Lorem Ipsum'|uppercase
        # final inline comment
    }}
