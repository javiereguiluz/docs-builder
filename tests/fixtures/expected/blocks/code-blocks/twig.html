<div translate="no" data-loc="9" class="notranslate codeblock codeblock-length-sm codeblock-twig">
    <div class="codeblock-scroll">
        <pre class="codeblock-lines">1
2
3
4
5
6
7
8
9</pre>
        <pre class="codeblock-code">
            <code>
                <span class="hljs-comment">{# some code #}</span>
                <span class="xml"></span>
                <span class="hljs-template-tag">
                    {%
                        <span class="hljs-name"><span class="hljs-keyword">set</span></span> some_var = 'some value' <span class="hljs-comment"># some inline comment</span>
                    %}
                </span>
                <span class="xml"></span>
                <span class="hljs-template-variable">{{
                    <span class="hljs-comment"># another inline comment</span>
                    'Lorem Ipsum'|uppercase
                    <span class="hljs-comment"># final inline comment</span>
                }}</span>
            </code>
        </pre>
    </div>
</div>
